ci/jobs/scripts/integration_tests_configs.py

```diff
@@ -64,53 +64,54 @@ class TC:
+    # TC("test_random_inserts/", True, "no idea why i'm sequential"),
+    # TC("test_replicated_database/", True, "no idea why i'm sequential"),
     TC("test_server_overload/", True, "no idea why i'm sequential"),
-    TC("test_server_reload/", True, "no idea why i'm sequential"),
-    TC("test_storage_kafka/", True, "no idea why i'm sequential"),
-    TC("test_storage_kerberized_kafka/", True, "no idea why i'm sequential"),
-    TC("test_storage_rabbitmq/", True, "no idea why i'm sequential"),
-    TC("test_storage_s3_queue/", True, "no idea why i'm sequential"),
-    TC("test_system_flush_logs/", True, "no idea why i'm sequential"),
-    TC("test_system_logs/", True, "no idea why i'm sequential"),
-    TC("test_user_ip_restrictions/", True, "no idea why i'm sequential"),
+    # TC("test_server_reload/", True, "no idea why i'm sequential"),
+    # TC("test_storage_kafka/", True, "no idea why i'm sequential"),
+    # TC("test_storage_kerberized_kafka/", True, "no idea why i'm sequential"),
+    # TC("test_storage_rabbitmq/", True, "no idea why i'm sequential"),
+    # TC("test_storage_s3_queue/", True, "no idea why i'm sequential"),
+    # TC("test_system_flush_logs/", True, "no idea why i'm sequential"),
+    # TC("test_system_logs/", True, "no idea why i'm sequential"),
+    # TC("test_user_ip_restrictions/", True, "no idea why i'm sequential"),
     TC(
         "test_backup_restore_on_cluster/test_concurrency.py",
         True,
         "no idea why i'm sequential",
     ),
-    TC("test_database_delta/test.py", True, "no idea why i'm sequential"),
-    TC("test_keeper_ipv4_fallback/test.py", True, "no idea why i'm sequential"),
-    TC("test_storage_iceberg_no_spark/", True, "no idea why i'm sequential"),
-    TC("test_storage_iceberg_with_spark/", True, "no idea why i'm sequential"),
+    # TC("test_database_delta/test.py", True, "no idea why i'm sequential"),
+    # TC("test_keeper_ipv4_fallback/test.py", True, "no idea why i'm sequential"),
+    # TC("test_storage_iceberg_no_spark/", True, "no idea why i'm sequential"),
+    # TC("test_storage_iceberg_with_spark/", True, "no idea why i'm sequential"),
     TC("test_storage_iceberg_with_spark_cache/", True, "no idea why i'm sequential"),
-    TC(
-        "test_storage_iceberg_schema_evolution/",
-        True,
-        "no idea why i'm sequential",
-    ),
-    TC("test_disks_app_interactive/", True, "no idea why i'm sequential"),
-    TC("test_storage_delta_disks/", True, "no idea why i'm sequential"),
-    TC("test_storage_iceberg_disks/", True, "no idea why i'm sequential"),
+    # TC(
+    #     "test_storage_iceberg_schema_evolution/",
+    #     True,
+    #     "no idea why i'm sequential",
+    # ),
+    # TC("test_disks_app_interactive/", True, "no idea why i'm sequential"),
+    # TC("test_storage_delta_disks/", True, "no idea why i'm sequential"),
+    # TC("test_storage_iceberg_disks/", True, "no idea why i'm sequential"),
```

Comment:
Should we cleanup this list? `no idea why i'm sequential` does not adds any extra context